README.md

```diff
@@ -86,14 +82,13 @@ If you do not want Terraform to create a new IAM user or you do not have permiss
 module "eks-jx" {
   source  = "jenkins-x/eks-jx/aws"
 
-  output "jx_requirements" {
```

Comment:
so output works, the problem is that module "eks-jx" has a missing `}`, before output.

Reply:
Output doesn't work, that was the subject of #100 
```
An output value with the name "jx_requirements" has not been declared in
module.eks-jx.
```

Reply:
I'm confused.  I tried with 1.2.0 and it works.  So at some point, I was looking at the README for 1.2.0 and using 1.1.0, I guess?  I'll update.

Reply:
(tidied up some of the spacing)

Reply:
Yeah, this was added in [v1.2.0](https://github.com/jenkins-x/terraform-aws-eks-jx/releases/tag/v1.2.0)